### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by PanwarJayant**

### CODE REVIEW

File: .github/workflows/deployment-script.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
             node-version: '12'
             check-latest: true
     - run: |
-        git clone -b pipeline-new https://github.com/virtual-labs/ph3-lab-mgmt
+        git clone https://github.com/virtual-labs/ph3-lab-mgmt
```
Comment (raj-vlabs) [inline .github/workflows/deployment-script.yml]: Please use --depth=1 to clone only the latest code. Look here:
https://github.com/virtual-labs/exp-atterberg-limits-iiith/blob/main/.github/workflows/deployment-script.yml

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (raj-vlabs) [inline .vscode/settings.json]: This file should be put in .gitignore

File: experiment/experiment-name.md [hunk 3]
```diff
@@ -1 +1 @@
-## Experiment name
+### Instrumentation and Working Principles of Solutions Infra Red (IR) Spectroscopy
```
Comment (raj-vlabs) [inline experiment/experiment-name.md]: Should Infrared be one word or Infra Red?

File: experiment/posttest.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        "question": " The O-H group in alcohol and carboxylic acids show at:",
+        "answers": {
+            "a": " Alcohol at 3300cm^-1 and carboxylic acid at 3000cm^-1",
+            "b": " At the same region around 3000cm^-1",
+            "c": " Alcohol at 3000cm^-1 and carboxylic acid at 3300cm^-1",
+            "d": " Cannot say"
+        },
+        "correctAnswer": "a"
+    }
```
Comment (raj-vlabs) [inline experiment/posttest.json:71]: Is it possible to have explanations to the answers?

File: experiment/theory.md [hunk 5]
```diff
@@ -1 +1,46 @@
-### Link your theory in here
\ No newline at end of file
+When two atoms combine to form a stable covalent moleculer, there are two repulsion forces acting between the two hetero atoms. One between the positively charged nuclei of both the atoms and the other between the negative electron clouds. The other force is the attraction between the nucleus of one atom with the electons of the other atom. Balancing the forces between them, the two atoms settle at a mean internuclear distance or the bond length where the total energy of the system is minimum. Any change like pulling the atoms away or squeezing them brings change in the bond length which requires an input of energy.
```
Comment (raj-vlabs) [inline experiment/theory.md]: The Theory has multiple grammatical mistakes, it needs to be reviewed by an SME

File: experiment/simulation/index.html [hunk 6]
```diff
@@ -1,13 +1,339 @@
...
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.6-b/css/style.min.css">
+		<link rel="stylesheet" href="./css/main.css">
+		<script
+      src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js"
+      integrity="sha512-z4OUqw38qNLpn1libAN9BsoDx6nbNFio5lA6CuTp9NlK83b89hgyCVq+N5FdBJptINztxn1Z3SaKSKUS5UP60Q=="
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://canvasjs.com/assets/script/canvasjs.min.js"></script>
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Please fix the indentation of the above few lines.

File: experiment/posttest.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
+[{
```
Comment (raj-vlabs) [inline experiment/posttest.json]: Are pretest and posttest going to stay in version 1?

File: experiment/simulation/index.html (hunk 6 above)
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Canvasjs needs license to be used. Please replace with plotly

File: experiment/simulation/index.html [hunk 8]
```diff
@@ -1,13 +1,339 @@
...
+                </svg>
+              </div>
+              
+            
+            <div class="apparatus-desiccator column" id="apparatus-desiccator">
+              <div class="row desiccator-row" id="desiccator-row">
+                <figure>
+                  <img
+                    src="./images/desiccator.png"
+                    alt="This is an animated gif image"
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Please change the alt attribute to describe the image meaningfully. Web crawlers use the alt attribute to understand an image.

File: experiment/simulation/js/main.js [hunk 9]
```diff
@@ -1 +1,610 @@
...
+        translateX: "500%",
+        translateY: "175%",
+      })
+      .add({
+        delay: "800",
+        rotateZ: "45",
+      })
+      .add({
+        update: function (anim) {
+          document.getElementById("spoon-mouth").style.fill = "#b83dba";
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: Please take the colors out in a separate variable or file with mnemonic keys and hexcode as values. Please take a look at https://github.com/virtual-labs/exp-atterberg-limits-iiith/blob/main/experiment/simulation/js/data.json

File: experiment/simulation/index.html [hunk 10]
```diff
@@ -1,13 +1,339 @@
...
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.6-b/css/style.min.css">
+		<link rel="stylesheet" href="./css/main.css">
+		<script
+      src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js"
+      integrity="sha512-z4OUqw38qNLpn1libAN9BsoDx6nbNFio5lA6CuTp9NlK83b89hgyCVq+N5FdBJptINztxn1Z3SaKSKUS5UP60Q=="
+      crossorigin="anonymous"
+    ></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Why do we need jQuery?

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
1. Please ensure to fix indentation on all files.
2. Please see if the pretest and posttest can be converted to the quiz 2.0 format.
3. canvasjs is a licensed library, it cannot be used without a license.
4. Multiple grammatical mistakes in many files. Please get them reviewed by the SME

File: experiment/simulation/index.html (hunk 6 above)
Comment (PanwarJayant) [reply experiment/simulation/index.html]: Replaced with Plotly

File: experiment/simulation/index.html (hunk 10 above)
Comment (PanwarJayant) [reply experiment/simulation/index.html]: Was testing something. Forgot to remove it. Removed now.

File: .vscode/settings.json (hunk 2 above)
Comment (PanwarJayant) [reply .vscode/settings.json]: Removed the unnecessary file.

File: experiment/theory.md (hunk 5 above)
Comment (PanwarJayant) [reply experiment/theory.md]: SME still working on the Theory part. Sent the questions for now. Will be adding those questions to the pretest and the postest.

File: experiment/objective.md [hunk 11]
```diff
@@ -0,0 +1,3 @@
+(i) To learn solution sample prepartion and handling procedures to measure an IR spectrum
```
Comment (raj-vlabs) [inline experiment/objective.md]: Move the objectives to the Aim file and remove hte objective.md file

File: experiment/objective.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
...
+
+(ii) Spectral analysis of the sample measured
```
Comment (raj-vlabs) [inline experiment/objective.md]: Keep both aims in the same tone. Start both with 'To ...'

**Review by raj-vlabs — CHANGES_REQUESTED** [review]

File: experiment/posttest.json [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+      },
+      "difficulty": "intermediate",
+      "correctAnswer": "a"
+    },
+    {
+      "question": "4. Which of the following molecules interferes with the recorded IR spectra when recorded in air?",
+      "answers": {
+        "a": "N&#8322",
+        "b": "O&#8322",
+        "c": "CO&#8322",
```
Comment (raj-vlabs) [inline experiment/posttest.json]: Why are we using this complex way of specifying a subscript?

**Review by raj-vlabs — APPROVED** [review]
Approving with minor comments. Please resolve them before merging.

**Pull request merged**